prussia_rt/src/routines.S

```diff
@@ -1,6 +1,7 @@
 # The assembler tries to reorder our instructions around delay slots.
 # Since we want to preserve this order, disable the reordering feature.
 .set noreorder
+.set noat
```

Comment:
I don't think I smash `$at` in my code, do I?

Reply:
My bad, removed.